### PR TITLE
Minor SLEIGH tweaks

### PR DESCRIPTION
**Pull request by mumbel**
* LDM and STM use table (removed hardcoded files)
* branch use table
* semicolon syntax cleanup

### CODE REVIEW

**Comment by mumbel** [comment]
The biggest change is the LDM and STM
```
        8f 38           STM1       (R10,R11,R12)
                                                      SP = INT_SUB SP, 4:4
                                                      STORE ram(SP), R12
                                                      SP = INT_SUB SP, 4:4
                                                      STORE ram(SP), R11
                                                      SP = INT_SUB SP, 4:4
                                                      STORE ram(SP), R10
        8d 1c           LDM1       (R10,R11,R12)
                                                      R10 = LOAD ram(SP)
                                                      SP = INT_ADD SP, 4:4
                                                      R11 = LOAD ram(SP)
                                                      SP = INT_ADD SP, 4:4
                                                      R12 = LOAD ram(SP)
                                                      SP = INT_ADD SP, 4:4
```

The branches should be the same, just used a table to reduce code duplication, hopefully, I did the logic slightly different, but it looks correct I think.

I also changed the delay slot instructions from  `_D` to `:D` and the LDI to use that as well (`LDI_32`->`LDI:32`)

**Comment by mumbel** [comment]
oh one syntax error in `INT` to build for 10.2

File: data/languages/fr60.sinc [hunk 1]
```diff
@@ -506,15 +684,15 @@ DIR8B_REL: reloc is dir8 [ reloc = dir8 * 1; ] {
 
 # ====
 
-:LDI_32 #i32, ri48_4 is op48_12=0x9f8 & i32 & ri48_4 {
+:LDI^":32" #i32, ri48_4 is op48_12=0x9f8 & i32 & ri48_4 {
```
Comment (DiscoStarslayer) [inline data/languages/fr60.sinc:687]: Thanks for introducing me to `^`. Helps keeps things aligned with the datasheets.

**Review by DiscoStarslayer — APPROVED** [review]
Branches look good, as do the delay slots and SDM/LDM! So much cleaner! Thank you for the contribution!

**Pull request merged**